### PR TITLE
Add plain text and html password reset email

### PR DESCRIPTION
**Pull request by mrchrisadams**
@antony, I’ve now pushed plain text and html versions of the password reset-email template.

These are based on Zurb's html email templates, because they have the best combination of [documentation](http://foundation.zurb.com/emails/docs.html), [examples to work from](http://foundation.zurb.com/emails/email-templates.html), and tests in all the various email clients.

We'll need to inline-all the styles into this template before it's ready to send - the fastest way I can think of doing this is to use their [Inliner CSS-inlining tool](http://foundation.zurb.com/emails/inliner.html), 

I've tried to be conservative with styles but I'll still need to check how outlook shows this.

### CODE REVIEW

File: resources/reset-your-password.html [hunk 1]
```diff
@@ -0,0 +1,883 @@
...
+<h2>Reset your password <small>Forgot your password? Don't worry, it happens to all of us.</small></h2>
+
+<p></p></center><p></p>
+
+<p>We've received a request to reset your password for using Witan. If you follow the link below, you can reset your password.</p>
+
+<p>Once you have done this, you can then sign in with your email address, and your new password, and be back on your way.</p>
+
+<p></p><table class="button radius"><tbody><tr><td>
+<a href="http://{{ link_to_reset_password  }}" title="Link to Google">Reset my password for Witan</a>
```
Comment (acron0) [inline resources/reset-your-password.html]: `Link to Google` should be changed

**Comment by acron0** [comment]
Also, could we move these into a 'templates' sub-dir please?

**Comment by mrchrisadams** [comment]
Okay, templates moved, and link title updated.

**Comment by acron0** [comment]
:shipit:

**Pull request merged**